package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@fastify/one-line-logger",
+  "version": "1.0.0",
+  "description": "A simple formatter for fastify's logs",
+  "main": "dist/index.js",
+  "types": "dist/index.d.js",
+  "directories": {
+    "doc": "docs",
+    "example": "examples",
+    "test": "test"
+  },
+  "dependencies": {
+    "pino-pretty": "^8.1.0"
+  },
+  "devDependencies": {
+    "@types/jest": "^28.1.6",
+    "@types/pino-pretty": "^4.7.5",
+    "@types/readable-stream": "^2.3.14",
+    "@typescript-eslint/eslint-plugin": "^5.33.0",
+    "@typescript-eslint/parser": "^5.33.0",
+    "eslint": "^8.21.0",
+    "eslint-config-standard": "^17.0.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^26.8.2",
+    "eslint-plugin-n": "^15.2.4",
+    "eslint-plugin-promise": "^6.0.0",
+    "fastify": "^4.3.0",
+    "jest": "^28.1.3",
+    "jest-runner-tsc": "^1.6.0",
+    "rimraf": "^3.0.2",
+    "ts-jest": "^28.0.7",
+    "ts-node": "^10.9.1",
+    "typescript": "<4.5.0"
+  },
+  "scripts": {
+    "test": "npm run build && npm run test:coverage",
+    "test:coverage": "jest --coverage",
+    "test:watch": "jest --watch --coverage",
+    "lint": "eslint \"*/**/*.{js,ts,tsx}\"",
+    "lint:fix": "eslint \"*/**/*.{js,ts,tsx}\" --fix",
+    "build": "rimraf ./dist && mkdir dist && tsc"
```

Comment:
Are we able to use plain JavaScript so that we don't need to build things? We will also be limiting who is able to contribute to this project by using a non-standard language.

Reply:
Will use JS. In general how do we decide if we are using TS or JS for a project?

Reply:
I don't think we have any TS based projects in the Fastify org. A significant portion of the team does not know TS.

Reply:
FYI, `fastify-passport` and `fastify-flash` are written in TypeScript.